packages/plugins/users-permissions/admin/src/translations/en.json

```diff
@@ -12,8 +12,8 @@
   "EditForm.inputToggle.label.email-confirmation-redirection": "Redirection url",
   "EditForm.inputToggle.label.email-reset-password": "Reset password page",
   "EditForm.inputToggle.label.sign-up": "Enable sign-ups",
-  "EditForm.inputToggle.placeholder.email-confirmation-redirection": "ex: https://yourfrontend.com/confirmation-redirection",
-  "EditForm.inputToggle.placeholder.email-reset-password": "ex: https://yourfrontend.com/reset-password",
+  "EditForm.inputToggle.placeholder.email-confirmation-redirection": "ex: https://yourfrontend.example.com/confirmation-redirection",
+  "EditForm.inputToggle.placeholder.email-reset-password": "ex: https://yourfrontend.example.com/reset-password",
```

Comment:
Using random domains one doesn't control is a bad practice. (Happy to drop if people really insist.)

Reply:
Can you do it in the other translations as well?